src/cli.rs

```diff
@@ -51,6 +51,10 @@ enum Command {
         base_url: String,
         #[clap(long, env, default_value = "false")]
         demo_mode: bool,
+        #[clap(long, env)]
```

Comment:
I think both of these flags should be optional - I'd like to support deployments that don't use Google for SSO.